core/src/main/kotlin/com/justai/jaicf/builder/ScenarioBuilder.kt

```diff
@@ -117,6 +120,37 @@ abstract class ScenarioBuilder(
         currentState = statesStack.last
     }
 
+    /**
+     * Appends an any-intent state to the scenario.
+     * This state will be activated for every request that has any intent in its input, recognized by
+     * any registered IntentActivator.
+     *
+     * The current dialogue's context won't be changed.
+     * This builder requires an IntentActivator to be added to the activators list of your BotEngine instance.
+     *
+     * ```
+     * anyIntent {
+     *   activator.caila?.topIntent?.answer?.let {
+     *       reactions.say(it)
+     *   }
+     * }
+     * ```
+     *
+     * @param state an optional state name ("anyIntent" by default)
+     * @param action an action block that will be executed
+     */
+    fun anyIntent(
```

Comment:
@nikvoloshin Good enough for the very first implementation. We could think about more general `unhandled` activation, that means "Activate on any input including _intent_, _event_ or _anything else_ that cannot be handled by any other state". How do you think?

Reply:
As far as I understand, it's exactly what `catchAll` activator means, isn't is?
If so, does that mean that we want to unify `catchAll`, `anyIntent` or anything else with similar logic to one `unhandled`-like rule?